test/Analyzers.Tests/GrainInterfaceMethodReturnTypeDiagnosticAnalyzerTest.cs

```diff
@@ -74,6 +74,8 @@ public async Task StaticInterfaceMethodsWithRegularReturnsAreAllowed()
                     public interface I : Orleans.IGrain
                     {
                         public static int GetSomeOtherThing(int a) => 0;
+                        public static virtual int GetSomeOtherThingVirtual(int a) => 0;
+                        public static abstract int GetSomeOtherThingAbstract(int a);
```

Comment:
Abstract members won't work because we cannot sensibly generate `GrainReference` classes which implement these members.

Reply:
Oh, sure! Thanks for pointing out.

Reply:
But as this has nothing to do with the purpose of the particular analyzer, I will only remove the abstract methods from the test cases but leave the analyzer as is (handling static and abstract static the same). If we'd restrict the analyzer to non-abstract, it would still allow something like `static abstract Task<int> X();`. So the check for static abstract should potentially be checked in a separate analyzer (that I won't be able to do though).